README.md

```diff
@@ -1,11 +1,11 @@
 # Microverse Leaderboard
 
-This is a javascript project application
+This is a javascript Leaderboard application
 
 Includes the following features...
 
```

Comment:
- [ ] **Optional:**
- Kindly add more sections to your README in future projects. Kindly [see this template](https://github.com/microverseinc/readme-template) for more sections. These give a more professional look to your project.